bin/cml/comment/create.js

```diff
@@ -18,21 +18,23 @@ exports.builder = (yargs) =>
     .options(exports.options);
 
 exports.options = kebabcaseKeys({
+  target: {
+    type: 'string',
+    description:
+      'Forge object to create comment on, can be one of pr, commit or issue. ' +
+      "Specify 'issue#123' to create a comment on a specific issue. " +
```

Comment:
what syntaxes are supported?

- `123`
- `#123`
- `issue#123`
- `pr#123`
- `mr#123`

Reply:
Valid targets are:
- `auto` determine comment target automatically:
  - PR if running in a PR context
  - PR if HEAD commit is part of a PR
  - HEAD commit (fallback)
- `pr` - determine pr by context or head commit (making allowances for how github's PR merge commits)
- `pr#123` - specific PR number
- `commit` - current HEAD commit
- `commit#adfa12` - specific commit
- `issue#123` - specific issue

Reply:
I think we need:

- `#123` to auto-infer pr-or-issue
- `mr` alias for `pr`
- `mr#123` alias for `pr#123`
- why do we have `commit#d34db3`? Doesn't `--commit-sha=d34db3` already cover this?
- full URL (e.g. if they want to target another PR returned by `cml pr`)